service/service.go

```diff
@@ -137,6 +162,20 @@ func New(config Config) (*Service, error) {
 // Boot starts top level service implementation.
 func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
-		go s.releaseController.Boot(context.Background())
+		ctx := context.Background()
+		backOff := backoff.NewMaxRetries(7, 1*time.Second)
+
+		// Install Release CRD.
+		s.logger.LogCtx(ctx, "level", "debug", "message", "ensuring release custom resource definition exists")
+
+		err := s.crdClient.EnsureCreated(ctx, s.crd, backOff())
```

Comment:
If we are going to install `Release` CRD in below link, https://github.com/giantswarm/operatorkit/blob/a1aa0b72cce3bb752ca8e5097dc84f97cf4cbbf7/controller/controller.go#L367
(operator-kit, controller module) 
Why we need to do that again in here?

Reply:
This is about installing `Release` CRD and not `ReleaseCycle` CRD